thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import peakdetection as pkd
+from IPython import embed
+
+
+def type_detector(data, samplerate, thres=0.1, plot_data_func=None):
+
+    """ Detects if fish is pulse or wave by calculating the proportion of the time distance between a peak and its
+     following trough, relative to the time between consecutive peaks.
+
+    :param data: (1-D array) The data to be analyzed (Usually the best window already)
+    :param samplerate: (float). Sampling rate of the data in Hz
```

Comment:
In my load functions I always call this just rate. We should decide for one of the terms. I am willing to change it to samplerate, but rate is nice and short.